src/app/component/eco-news/components/filter-news/filter-news.component.ts

```diff
@@ -35,14 +35,26 @@ export class FilterNewsComponent implements OnInit {
   public toggleFilter(currentFilter: string): void {
     this.filters.map(el => el.isActive = el.name === currentFilter ? !el.isActive : el.isActive);
     this.emitActiveFilters();
+    this.setSessionStorageFilters();
   }
 
   private setTags(tags: Array<string>): void {
-    this.filters = tags.map((filter: string) => ({name: filter, isActive: false}));
+    const storageFilters = this.getSessionStorageFilters();
+    this.filters = tags.map((filter: string) => ({ name: filter, isActive: storageFilters.find(el => el === filter) !== undefined }));
```

Comment:
correct way to test undefined variable is to use `typeof`

Reply:
fixed